generators/client/files-angular.js

```diff
@@ -128,7 +128,6 @@ const files = {
             path: ANGULAR_DIR,
             templates: [
                 '_app.main.ts',
-                '_app.main-aot.ts',
```

Comment:
I just upgraded one of my project, and I have this error:
```
ERROR in [at-loader] ./src/main/webapp/app/app.main-aot.ts:3:44 
    TS2307: Cannot find module '../../../../target/aot/src/main/webapp/app/app.module.ngfactory'.
error Command failed with exit code 2.
```

As this file is deleted, shouldn't `app.main-aot.ts` be in `cleanup.js` ?